Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -377,6 +379,19 @@ void HexEditorWidget::update_inspector_values(size_t position)
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::UTF16, "");
     }
 
+    StringBuilder ascii_string_builder;
+    for (size_t i = 0; i < selected_bytes.size(); i++) {
+        char byte_char = static_cast<char>(selected_bytes[i]);
+
+        if (!is_ascii_control(byte_char) && is_ascii(byte_char)) {
+            ascii_string_builder.append(byte_char);
+        } else {
+            ascii_string_builder.append(" ");
+        }
```

Comment:
no curlies

Reply:
Oh, cleaning up the git commit history... I see what you are saying. Yeah, let me look into that...

Reply:
I condensed both commits together, so this issue is now defunct/superfluous